server/services/interventionsService.test.ts

```diff
@@ -2846,6 +2846,60 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
   */
+
+  describe('getServiceProviderReportingData', () => {
+    const reportingResponse = [
+      {
+        referralLink: 'https://refer-and-monitor.com/referral/c9f9e22f-ddd9-422a-a9af-76fc02b18b38',
```

Comment:
We have the opportunity to simplify by not expecting this from the API

_nit_ it's recommended to use `domain.example` and similar hostnames in case it ever gets called by accident (https://en.wikipedia.org/wiki/.example)

Reply:
It's a little more fiddly on the UI side to construct this due to the way we rely on Typescript assertions to essentially deserialize our objects for us. I can have a go, but I feel it's a bit simpler to send this in the response body so we can pipe the whole response straight into a CSV, rather than prepending this field. What do you think?

In the meantime, I've updated the link with `.example`.